queries/scss/highlights.scm

```diff
@@ -29,6 +33,9 @@
 (each_statement (value) @parameter)
 (each_statement (variable_value) @parameter)
 
+(for_statement (variable) @parameter)
+(for_statement (_ (variable_value) @parameter))
+
 (argument) @parameter
 (arguments (variable_value) @parameter)
```

Comment:
not sure why parameter is used for loops and arguments in this file. might have it's reasons

Reply:
Not familiar enough with the conventions to have any opinion about this. Anything you want me to change?

Reply:
Usually `@parameter` only applies to the parameters in function declarations. Not sure how they are used here in scss. Overall your PR looks good.